### ISSUE TITLE
TBC Issue

### ISSUE THREAD

**Issue by Glaived** [Tojaso/Buffle#3]
https://github.com/Tojaso/Bufflehead/blob/d6449d0d6fea1199b3554078cebead906844d94c/Main.lua#L348

```
5x Bufflehead\Main.lua:348: attempt to index global 'C_PetBattles' (a nil value)
[string "@Bufflehead\Main.lua"]:348: in function `CheckBlizzFrames'
[string "@Bufflehead\Main.lua"]:269: in function `?'
[string "@Auc-Advanced\Libs\LibDataBroker\CallbackHandler\CallbackHandler-1.0-7.lua"]:119: in function <...ibDataBroker\CallbackHandler\CallbackHandler-1.0.lua:119>
[string "=[C]"]: ?
[string "@Auc-Advanced\Libs\LibDataBroker\CallbackHandler\CallbackHandler-1.0-7.lua"]:29: in function <...ibDataBroker\CallbackHandler\CallbackHandler-1.0.lua:25>
[string "@Auc-Advanced\Libs\LibDataBroker\CallbackHandler\CallbackHandler-1.0-7.lua"]:64: in function `Fire'
[string "@Bagnon\common\Wildpants\libs\AceEvent-3.0\AceEvent-3.0-4.lua"]:120: in function <...\common\Wildpants\libs\AceEvent-3.0\AceEvent-3.0.lua:119>
```

![image](https://user-images.githubusercontent.com/1835170/118850485-1c746d80-b8d1-11eb-9299-6578bcb508c6.png)

**Comment by Tojaso** [comment]
I have a fix in the Raven version for TBC and am in the process of adding similar fix to Bufflehead. Should be fairly soon.

**Comment by Tojaso** [comment]
Fixed in v1.0.3.1. This version also no longer requires separate builds for different serves so it will show as up-to-date on classic, TBC and retail.
